data/json/monsters.json

```diff
@@ -1033,7 +1033,7 @@
     "id": "mon_eyebot",
     "type": "MONSTER",
     "name": "eyebot",
-    "description": "A fusion-driven UAV largely comprised of a high-resolution camera lens and a speaker, this spheroid robot hovers above the ground, documenting the carnage and mayhem around it.",
+    "description": "A fusion-driven quadcopter drone largely comprised of a high-resolution camera lens and a speaker, this robot hovers above the ground, documenting the carnage and mayhem around it.  While it is no longer able to report to a central authority, it's likely that some of its automated alert systems are still functional, given that the thing seems to have an operational charging station somewhere.",
```

Comment:
```suggestion
    "description": "A quadcopter drone largely comprised of a high-resolution camera lens and a speaker, this robot hovers above the ground, documenting the carnage and mayhem around it.  While it is no longer able to report to a central authority, it's likely that some of its automated alert systems are still functional, given that the thing seems to have an operational charging station somewhere.",
```

Reply:
@kevingranade you overwrote the quadcopter part.